tests/unit-core/test-api.c

```diff
@@ -255,8 +257,7 @@ foreach (const jerry_value_t name, /**< field name */
   }
   else if (!strncmp (str_buf_p, "bravo", (size_t) sz))
   {
-    TEST_ASSERT (jerry_value_is_boolean (value));
-    TEST_ASSERT (jerry_get_boolean_value (value) == false);
+    TEST_ASSERT (jerry_value_is_false(value));
```

Comment:
jerry_get_boolean_value is also removed here and below, please restore all. and also add a space before open bracket